Source/WebKit/UIProcess/ProcessThrottler.h

```diff
@@ -70,7 +70,7 @@ class ProcessThrottlerActivity : public RefCountedAndCanMakeWeakPtr<ProcessThrot
             invalidate(ForceEnableActivityLogging::No);
     }
 
-    bool isValid() const { return !!m_throttler; }
+    inline bool isValid() const { return !!m_throttler; }
```

Comment:
I don't think this is doing anything. You need to make the change in ProcessAssertion, no?

Reply:
Agreed this doesn't do anything, please revert changes to this file as well.